config/locales/de.default.yml

```diff
@@ -1,6 +1,9 @@
 ---
 de:
   activerecord:
+    request:
+        one: "Anfrage"
+        other: "Anfragen"
```

Comment:
This isn't needed, is it? The translation in duplicated in `config/locales/de.requests.yml`